extensions/ql-vscode/src/vscode-tests/minimal-workspace/index.ts

```diff
@@ -1,14 +1,8 @@
 import "source-map-support/register";
-import * as sinonChai from "sinon-chai";
-import * as chai from "chai";
 import "chai/register-should";
-import * as chaiAsPromised from "chai-as-promised";
 
 import { runTestsInDirectory } from "../index-template";
 
-chai.use(chaiAsPromised);
-chai.use(sinonChai);
-
 export function run(): Promise<void> {
```

Comment:
I believe this complete file can be deleted, not just the things related to chai/sinon.